pyqt-apps/siriushla/si_ap_fofb/trig_acq_config.py

```diff
@@ -172,10 +183,10 @@ def _basicSettingsWidget(self):
         lay.addWidget(self.lb_rep, 3, 2)
         lay.addWidget(ld_nrpre, 4, 0)
         lay.addWidget(self.sb_nrpre, 4, 1)
-        lay.addWidget(self.lb_nrpre, 4, 2)
+        lay.addWidget(self.fr_nrpre, 4, 2)
         lay.addWidget(ld_nrpos, 5, 0)
         lay.addWidget(self.sb_nrpos, 5, 1)
-        lay.addWidget(self.lb_nrpos, 5, 2)
+        lay.addWidget(self.fr_nrpos, 5, 2)
```

Comment:
Would it be simple to have this alarm frame be around both of the sample readbacks, instead of being around each individual readback?

Reply:
done!